packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -60,30 +76,39 @@ const BLMAlert: React.FC = () => {
   );
 };
 
+const PasteThemeAlert: React.FC<{isPasteTheme: boolean}> = ({isPasteTheme}) => {
+  if (!isPasteTheme) {
+    return null;
+  }
+
+  return (
+    <Alert variant="warning">
+      <Text as="p">
+        <strong>WARNING:</strong> The Paste theme is an <em>extremely early</em> preview of future work!{' '}
+        <Anchor
+          href="https://docs.google.com/document/d/1H2Rj3NEmVSv0yxMBRjYOjruQllO__uj2-I_ibIoumZs/edit?usp=sharing"
+          target="_blank"
+        >
+          Read the FAQ
+        </Anchor>{' '}
+        for more information.
+      </Text>
+    </Alert>
+  );
+};
+
 export const SiteBody: React.FC = ({children}) => {
   const {theme: activeTheme} = useActiveSiteTheme();
+  const isPasteTheme = activeTheme === 'default';
   return (
     <>
-      <BLMAlert />
-      {activeTheme === 'default' && (
-        <Alert variant="warning">
-          <Text as="p">
-            <strong>WARNING:</strong> The Paste theme is an <em>extremely early</em> preview of future work!{' '}
-            <Anchor
-              href="https://docs.google.com/document/d/1H2Rj3NEmVSv0yxMBRjYOjruQllO__uj2-I_ibIoumZs/edit?usp=sharing"
-              target="_blank"
-            >
-              Read the FAQ
-            </Anchor>{' '}
-            for more information.
-          </Text>
-        </Alert>
-      )}
-      <StyledSiteBody activeTheme={activeTheme}>
+      <BlackLivesMatterAlert />
+      <PasteThemeAlert isPasteTheme={isPasteTheme} />
+      <StyledSiteBody isPasteTheme={isPasteTheme}>
         <Sidebar />
-        <Box height="100vh" overflow="auto">
+        <Box height="100%" overflow="auto">
           <SiteHeader />
-          <SiteMain id="site-main">
+          <SiteMain id="site-main" tabIndex={0} role="main">
```

Comment:
No need for role here, it is a `<main />` element that comes with an implicit `man` role.

Reply:
That isn't supported in IE11: https://www.drupal.org/project/drupal/issues/2862904#comment-12161565